### PR TITLE
loadpng, update old bep file

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: media-libs/loadpng/loadpng-1.5.recipe [hunk 1]
```diff
@@ -1,33 +1,48 @@
-DESCRIPTION="
-loadpng is some glue that makes it easy to use libpng to load and save bitmaps \
-from Allegro programs.
-"
+SUMMARY="Load and save bitmaps from Allegro programs"
+DESCRIPTION="loadpng is some glue that makes it easy to use libpng to load \
+and save bitmaps from Allegro programs."
 HOMEPAGE="http://tjaden.strangesoft.net/loadpng/"
```
Comment (alaviss) [inline media-libs/loadpng/loadpng-1.5.recipe]:
```diff
-HOMEPAGE="http://tjaden.strangesoft.net/loadpng/"
+HOMEPAGE="https://tjaden.strangesoft.net/loadpng/"

File: media-libs/loadpng/loadpng-1.5.recipe [hunk 2]
```diff
@@ -1,33 +1,48 @@
...
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:make
+	"
+
 BUILD()
 {
-	cd loadpng-1.5
-	make
+	make $jobArgs
```
Comment (alaviss) [inline media-libs/loadpng/loadpng-1.5.recipe]: It's also possible to build a shared lib, but I don't know if it's a good idea.

    make $jobArgs shared

File: media-libs/loadpng/loadpng-1.5.recipe [hunk 3]
```diff
@@ -1,33 +1,48 @@
...
+	lib:libz
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:liballeg
+	devel:libpng16
+	devel:libz
+	"
+
```
Comment (alaviss) [inline media-libs/loadpng/loadpng-1.5.recipe]:
```diff
-

File: media-libs/loadpng/loadpng-1.5.recipe (hunk 2 above)
Comment (Begasus) [reply media-libs/loadpng/loadpng-1.5.recipe]: There is an option to build a shared file, but even the author didn't wan't to use that as a default one?

**Pull request merged**